pandas/_libs/tslibs/offsets.pyx

```diff
@@ -375,3 +376,21 @@ class BaseOffset(_BaseOffset):
             # i.e. isinstance(other, (ABCDatetimeIndex, ABCSeries))
             return other - self
         return -self + other
+
+# ---------------------------------------------------------------------
+# Ticks
+
+class _Tick(object):
+    _inc = Timedelta(microseconds=1000)
+    _prefix = 'undefined'
```

Comment:
i would make this None as it would hard fail if not defined (or you can make it NotImemenrsError)